src/SocialPostManager.php

```diff
@@ -0,0 +1,280 @@
+<?php
```

Comment:
It seems that SocialAuthUserManager and SocialPostManager shares certain methods and attributes